app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/actions/AuthenticatedActionWithRetrievedData.scala

```diff
@@ -265,7 +265,7 @@ class AuthenticatedActionWithRetrievedData @Inject() (
           s"Agent has ${AgentsEnrolment.key} enrolment but does not have ${AgentsEnrolment.agentReferenceNumberIdentifier} identifier"
         )
       }
-      .sequence[Either[String, ?], AgentReferenceNumber]
+      .sequence[Either[String, *], AgentReferenceNumber]
```

Comment:
what's the difference between `*` and `?` here?

Reply:
? is deprecated now

Reply:
ah ok - I'll change them over if I ever come across any as well

Reply:
I think I did them all across the services.